package.json

```diff
@@ -268,6 +269,7 @@
     "postcss-loader": "2.1.4",
     "potools": "^0.2.0",
     "prettier": "1.13.6",
+    "pretty-quick": "1.6.0",
```

Comment:
We pinned `prettier` to an exact version so that all devs are using the same one. I guess we should do the same here which means you'll need to add this to the exception list in the test suite. The failing test should have info about it.